frontend/src/views/supp-allowances/SupplementaryAllowanceView.vue

```diff
@@ -1,3 +1,210 @@
 <template>
-  <h1>Supplementary Allowance</h1>
+  <v-container fluid>
+    <h1 class="my-2">Supplementary Allowance</h1>
+    <div v-if="loading" align="center">
+      <v-progress-circular indeterminate size="100" :width="6" color="#003366" class="min-height-screen"></v-progress-circular>
+    </div>
+    <div v-else>
+      <div class="min-height-screen my-4">
+        <v-row v-if="!$route.params.applicationGuid" no-gutters class="my-8">
+          <v-col cols="12" md="5" xl="3" class="pr-4">
+            <AppLabel>To start your application, select a facility:</AppLabel>
+            <div>
+              <v-icon class="mr-1">mdi-information-slab-circle-outline</v-icon>
+              <span>If your facility is not listed, contact your Account Manager.</span>
+            </div>
+          </v-col>
+          <v-col cols="12" md="7" xl="9">
```

Comment:
Can we limit the width of this to something reasonable?

Reply:
it is updated.